macros/base/start_print.cfg

```diff
@@ -393,3 +399,43 @@ gcode:
             ADAPTIVE_BED_MESH SIZE={FL_SIZE}
         {% endif %}
     {% endif %}
+
+
+[gcode_macro _MODULE_CUSTOM1]
+gcode:
+    # ---- CUSTOM Macro section
+    # this section is reserved for personal customized start actions, which can be combined with all other start actions
+    # in order to use this, create a new macro in overrides.cfg
+    # [gcode_macro _MODULE_CUSTOM1]
+    # renaming_existing: BASE_MODULE_CUSTOM1
```

Comment:
No rename existing is needed as it's not a standard Klipper macro. So if you put this line, it will error out I think (and even if it's not the case, it's not needed)